controllers/pkg/reconcilers/approval/reconciler.go

```diff
@@ -47,7 +47,7 @@ const (
 	DelayAnnotationName          = "approval.nephio.org/delay"
 	PolicyAnnotationName         = "approval.nephio.org/policy"
 	InitialPolicyAnnotationValue = "initial"
-	RequeueDuration              = 10 * time.Second
+	RequeueDuration              = 15 * time.Second
```

Comment:
@efiacor What does it signify? Should this value be statically calculated?

Reply:
How do we know that 15 is the right value?

Reply:
How would you propose that we statically calculate it?

Reply:
I don't know, how it is working or in the first place what are the factors which govern it. 
You changed it to 15 instead of 10 so I thought you would know the reason why exactly this value.

Reply:
> " I noticed intermittent issues where the AMF especially would go from approved - v1, to being reprocessed and a v2 branch getting created"

Do you know if it was the approval controller that approved the packagerevision prematurely, or was it some other actor?

Reply:
The only other actor would be the test script itself or a human. Tweaking timings is a losing battle, the overall flow should be robust to race conditions. So, I using a different approval policy makes more sense.

Reply:
Still doing some debugging to determine the cause/source but it seems the pv controller is triggering a new "clone" when it finds a diff in the KptFile.
https://github.com/nephio-project/porch/blob/main/controllers/packagevariants/pkg/controllers/packagevariant/packagevariant_controller.go#L827
This comes from the generic specializer controller I believe. 
Even with using an "always" policy I am seeing some odd behaviour. Will continue with more refinements later.